cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-database-syntax.asciidoc

```diff
@@ -1,72 +1,72 @@
 
 | [source, cypher]
 GRANT INDEX[ES] [MANAGEMENT]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to create and delete indexes on the database `name` or all databases
+| Enable the specified role or roles to create and delete indexes on the database `name` or all databases.
 
 | [source, cypher]
 GRANT {CREATE \| DROP} CONSTRAINT[S]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to create or delete indexes on the database `name` or all databases
+| Enable the specified role or roles to create or delete indexes on the database `name` or all databases.
 
 | [source, cypher]
 GRANT CONSTRAINT[S] [MANAGEMENT]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to create and delete constraints on the database `name` or all databases
+| Enable the specified role or roles to create and delete constraints on the database `name` or all databases.
 
 | [source, cypher]
 GRANT CREATE NEW [NODE] LABEL[S]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to create new labels for nodes in the database `name` or all databases
+| Enable the specified role or roles to create new labels for nodes in the database `name` or all databases.
 
 | [source, cypher]
 GRANT CREATE NEW [RELATIONSHIP] TYPE[S]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to create new types for relationships in the database `name` or all databases
+| Enable the specified role or roles to create new types for relationships in the database `name` or all databases.
 
 | [source, cypher]
 GRANT CREATE NEW [PROPERTY] NAME[S]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to create new names for properties in the database `name` or all databases
+| Enable the specified role or roles to create new names for properties in the database `name` or all databases.
 
 | [source, cypher]
 GRANT NAME [MANAGEMENT]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to create new labels, relationship types and property names in the database `name` or all databases
+| Enable the specified role or roles to create new labels, relationship types and property names in the database `name` or all databases.
 
 | [source, cypher]
 GRANT ALL [[DATABASE] PRIVILEGES]
-    ON DATABASE[S] {name \| *}
+    ON DATABASE[S] {* \| name}
     TO role[, ...]
-| Enable the specified role or roles to perform all of the above database actions on the database `name` or all databases
+| Enable the specified role or roles to perform all of the above database actions on the database `name` or all databases.
```

Comment:
```suggestion
| Enable the specified role or roles to perform all database privilege actions on the database `name` or all databases.
```

Reply:
I'm not sure that is clear/correct either, I'd prefer to just leave it for now and fix all places as part of https://trello.com/c/6sTonnfa/2296-dont-use-all-above-actions-for-all-privileges instead

Reply:
Ok, thank you.